index.js

```diff
@@ -0,0 +1,42 @@
+const testLineup = ['1B', '2B', '3B', 'C', 'OF', 'OF', 'OF', 'P', 'SS']
```

Comment:
have to make sure testLinup is always sorted too. better to sort it once by program on start rather manually by hand.

testLineup = testLineup.sort()